src/NuGet.Core/NuGet.Protocol/Plugins/UnixPlatformsEmbeddedSignatureVerifier.cs

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+
+namespace NuGet.Protocol.Plugins
+{
+    /// <summary>
+    /// Embedded Signature Verifier for the MacOS and Linux platforms.
+    /// </summary>
+    public class UnixPlatformsEmbeddedSignatureVerifier : EmbeddedSignatureVerifier
+    {
+        /// <summary>
+        /// Checks if a file has a valid embedded signature.
+        /// </summary>
+        /// <param name="filePath">The path of a file to be checked.</param>
+        /// <returns><c>true</c> if the file has a valid signature; otherwise, <c>false</c>.</returns>
+        /// <exception cref="ArgumentException">Thrown if <paramref name="filePath" />
```

Comment:
remove this as the implementation doesn't throw any exceptions

Reply:
the description is verbatim from the interface. 
It's probably a better idea to throw on null and empty strings actually :)
nice catch.